docs/source/models/cause.rst

```diff
@@ -41,29 +41,251 @@ Common data sources for cause models
+|Mortality     |                                |                    |
++--------------+--------------------------------+--------------------+
 
 Prevalence
 ++++++++++
 
-Cause-specific mortality
+Prevalence is defined as the **proportion of a given population that possesses a given condition or trait** at a 
+given timepoint.
+
+	For example, the prevalence of obesity in the United States was approximately 40% in 2016.
+
+When a time *frame* (such as 2016, i.e. 1/1/16 to 12/31/16) instead of a *timepoint* (such as 1/1/16) is 
+reported, it is commonly assumed that the reported prevalence represents the prevalence of the *midpoint* of 
+that timeframe (7/1/16 is the midpoint for the time frame of all of 2016). However, this may not always be the 
+case and it should be considered when relevant.
+
+Prevlance data can be used to **initialize cause model states** and represents the **probability that a simulant 
+will begin the simulation in a given state.**
+
+	For example, the probability that a simulant in a model of obesity in the United States beginning in 
+	2016 will begin the simulation as obese is 0.4 or 40%.
+
+Birth Prevalence
+++++++++++++++++
+
+Birth prevalence is defined as the **proportion of live births in a given population that possess a given 
+condition or trait at birth.**
+ 
+	For example, the birth prevalence for cleft lip in the United States in 2006 was 10.6 per 10,000 live 
+	births, or 0.106%.
+
+Birth prevalence data can be used to **initialize neonatal cause model states** and represent the **probability that a 
+simulant who is born during the simulation will be born into a given neonatal cause model state.** 
+
+	For example, the probability that a simulant born during a simulation of cleft lip in the United States 
+	in 2006 is 0.00106, or 0.106%.
+
+Incidence Rates
++++++++++++++++
+
+Incidence rates are defined as the **number of new cases of a condition that occur per person-year of the at-risk 
+population (individuals without condition).** 
+
+	For example, the incidence of multiple sclerosis (MS) in the United States is 2.8 per 100,000 
+	person-years. 
+
+	This suggests that if we followed 100,000 individuals without MS for 1 year each (100,000 people * 
+	1 year = 100,000 person-years), we would expect 2.8 of these individuals to develop MS within this timeframe. 
+
+	Alternatively, if we followed 50,000 individuals without MS for 2 years each (50,000 people * 2 years = 100,000 
+	person-years), we also would expect 2.8 of these individuals to develop MS within this timeframe.
+
+Incidence can be used to **estimate cause model transition rates** and can represent the **probability that a simulant 
+will transition from a susceptible state to an infected state within a given timestep.** 
+
+	For example, with a timestep of one year, the probability that a simulant will transition from a 
+	susceptible (without MS) cause model state to an infected (with MS) cause model state is 2.8*10^(-5).
```

Comment:
I think that this is an approximation of the precise probability, and the approximation is less accurate when the product is large.

Reply:
I made some changes that I hope make this more clear.

I was hoping that this example represented the "ideal" scenario in which the denominator actually represented the at-risk population, so the probability would not be an approximation. 

Then, I expanded the examples when the incidence rate would be an approximation of the transition rate below. 

Let me know if you think this still needs additional clarification after my updates. Thanks!

Reply:
Important clarifications, thanks.  There is still something else that I am confused about, which is a distinction between "instantaneous rates" and "annual rates".  But let's merge to keep things moving, and remember that there is still something I am confused about here.